validator/proposer/service.go

```diff
@@ -5,35 +5,43 @@ package proposer
 import (
 	"context"
 
+	pb "github.com/prysmaticlabs/prysm/proto/beacon/rpc/v1"
 	"github.com/prysmaticlabs/prysm/validator/types"
 	"github.com/sirupsen/logrus"
 )
 
 var log = logrus.WithField("prefix", "proposer")
 
+type rpcClientService interface {
```

Comment:
What is this and why can't you accept `pb.ProposerServiceClient` as an argument?

Reply:
It is the RPC service that runs in our repo. I cannot pass in the interface directly because it will not be defined upon service registration, only upon service `Start()`.

Reply:
Any potential solutions to this issue @prestonvanloon ?

Reply:
When is it set? I don't understand why this wrapper is necessary

Reply:
It is set upon the RPC service's .Start() function running

Reply:
We can discuss this in a separate PR or Issue - I'd rather get this one merged in soon as there will be substantial changes coming in